index.html

```diff
@@ -97,7 +97,12 @@ <h2>Security and Privacy Considerations</h2>
           The user visits the website or an <a>associated origin</a>.
         </li>
         <li>
-          The cookies expire or are deleted on the website or an <a>associated origin</a>.
+          If there is a <a>cookie name filter</a> that matches the name of the cookie
+          that changed.
+        </li>
+        <li>
+          If there is no <a>cookie name filter</a>. Then when any cookie expires or is
```

Comment:
nit: This can be one sentence. "If there is no cookie name filter, then when any cookie..."